### PR TITLE
Update git with latest internal version.

### PR DESCRIPTION
**Pull request by ZachMarcus**

### CODE REVIEW

File: src/quipper/perf_buildid.cc [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// Copyright 2016 The Chromium OS Authors. All rights reserved.
```
Comment (aalexand) [inline src/quipper/perf_buildid.cc:1]: 2016 is not right.

File: src/quipper/perf_buildid.h [hunk 2]
```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 The Chromium OS Authors. All rights reserved.
```
Comment (aalexand) [inline src/quipper/perf_buildid.h:1]: Ditto.

File: src/quipper/perf_buildid_test.cc [hunk 3]
```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2012 The Chromium OS Authors. All rights reserved.
```
Comment (aalexand) [inline src/quipper/perf_buildid_test.cc:1]: 2012 isn't right either I think.

File: src/quipper/perf_buildid.cc (hunk 1 above)
Comment (ZachMarcus) [reply src/quipper/perf_buildid.cc:1]: Good catch, will update those and overwrite this branch with both this change and an addition of authors to get past the CLA failures.

File: src/quipper/huge_page_deducer.cc [hunk 4]
```diff
@@ -359,12 +359,19 @@ void CombineMappings(RepeatedPtrField<PerfEvent>* events) {
 
     // Don't use IsEquivalentFile(); we don't want to combine //anon with
     // files if DeduceHugepages didn't already fix up the mappings.
-    should_merge = should_merge && prev_mmap->filename() == mmap->filename() &&
+    //
+    // TODO(b/169891636):  For hugetlbfs-backed files, We should verify that the
```
Comment (aalexand) [inline src/quipper/huge_page_deducer.cc:363]: nit: "We" -> "we"

Can be addressed later.

**Review by aalexand — APPROVED** [review]

**Pull request merged**